### ISSUE TITLE
amp-carousel-0.2 does not emit high trust actions on autoplay

### ISSUE THREAD

**Issue by jaygray0919** [ampproject/amphtml#24335]
Here is a carousel using `amp-carousel-0.2.js`. Progress status fails on `autoplay` although it works on manual.
  https://ampb.in/#3Ww9RcVJyVOWnhiqMy9y

With `amp-carousel-0.1.js` both work (`autoplay` and manual).
Toggle `ampb.in` to see the difference.
We don't want to use `amp-carousel-0.1.js` because of the lack of smooth scrolling provided by `amp-carousel-0.2.js`.
Is there a work around (or better coding) for progress status with `amp-carousel-0.2.js`?
We'll use an alternate approach if we can successfully echo progress through the carousel.

**Comment by sparhami** [comment]
This is due to #21171 being addressed in carousel 0.2, but not in carousel 0.1. Basically, we do not want to emit high trust actions from something like autoplay, since developers can use the action to do things that make the page jump.

We are working on a new gallery component that will have support for customizable progress indicators built into it.

There has been some discussion on whether or not we can allow some kind of changes on these events (e.g. changing colors, opacity, but not layout affecting things), but we don't have any solution at the moment.

/cc @nainar

**Comment by jaygray0919** [comment]
What is your advice?
-  Use `amp-carousel-0.1` as we have it now and wait for better transition in a new carousel release?
-  Remove `autoplay` from our current implementation and encourage the user to swipe thru the testimonials?
-  Something else?

**Comment by nainar** [comment]
Between options A and B - it depends on how your company sees it? Do you see a lot of users using the progress indicators to swipe through the testimonials? 

Also is there an A/B test that you can run for with and without progress indicator?

**Comment by caroqliu** [comment]
Closing this issue since it can now be resolved with [`<amp-inline-gallery>`](https://amp.dev/documentation/components/amp-inline-gallery/)

**Comment by jaygray0919** [comment]
Did not know about this lib - good to learn here!
